### PR TITLE
[[ Bug 20551 ]] Update Android Notification module to compile against API 23

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: engine/src/java/com/runrev/android/NotificationModule.java [hunk 1]
```diff
@@ -511,14 +500,42 @@ public static void showStatusBarNotification(Context context, int p_id, String p
...
+        t_builder.setSmallIcon(t_icon);
+        t_builder.setAutoCancel(true);
+        t_builder.setNumber(p_badge_value);
+        
+        if (p_play_sound)
+        	t_builder.setDefaults(Notification.DEFAULT_SOUND);
+        
+        t_builder.setContentIntent(createNotificationContentIntent(context));
+        t_builder.setContentText(p_body);
+        t_builder.setContentTitle(p_title);
```
Comment (livecodeali) [inline engine/src/java/com/runrev/android/NotificationModule.java:533]: You've set the title twice :-)

Comment (livecodeian) [reply engine/src/java/com/runrev/android/NotificationModule.java:533]: I've deleted the duplicate

Comment (livecodeali) [reply engine/src/java/com/runrev/android/NotificationModule.java:533]: Heh, great commit

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 637ebd6

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 637ebd651e915c5a4e731c248318b12495fa2133

**Comment by livecode-vulcan** [comment]
😞 test failure 637ebd651e915c5a4e731c248318b12495fa2133

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1495)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1511)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1485)
* try-community-universal-ios-iphoneos11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.4/builds/12)
* try-community-universal-ios-iphonesimulator11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.4/builds/12)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1489)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1515)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1495)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1509)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1494)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1504)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1544)
* try-community-x86_64-win32: [failure](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1484)

**Comment by livecodepanos** [comment]
@livecode-vulcan retry 637ebd6

**Comment by livecode-vulcan** [comment]
😎 test success 637ebd651e915c5a4e731c248318b12495fa2133

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1496)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1512)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1486)
* try-community-universal-ios-iphoneos11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.4/builds/13)
* try-community-universal-ios-iphonesimulator11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.4/builds/13)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1490)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1516)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1496)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1510)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1495)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1505)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1545)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1485)

**Pull request merged**